docs/subvolume.md

```diff
@@ -0,0 +1,55 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
```

Comment:
I think this tool uses health check user for the external ceph cluster, AFAIK that user will not have all the required access to do subvolume operations.